src/install.js

```diff
@@ -12,6 +12,7 @@ const util = require('util');
 const promptGet = util.promisify((schema, callback) => prompt.get(schema, callback));
 
 const utils = require('./utils');
+const { WSAEHOSTDOWN } = require('constants');
```

Comment:
Is this necessary?

Reply:
interesting... could this have been inserted automatically by `vscode`? (I certainly didn't add it personally). So I guess it's not necessary...